CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # UNRELEASED
-<!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->
-
+* Added full support for adding and updating custom permissions to `TeamProjectAccesses`. A `TeamProjectAccessType` of `"custom"` can set various permissions applied at
+the project level to the project itself (`TeamProjectAccessProjectPermissionsOptions`) and all of the workspaces in a project (`TeamProjectAccessWorkspacePermissionsOptions`) by @rberecka [#745](https://github.com/hashicorp/go-tfe/pull/745)<!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->
```

Comment:
<!-- Add CHANGELOG entry to this section for any PR awaiting the next release --> 
--> is this line supposed to above?

Reply:
I am referring to: <!-- Add CHANGELOG entry to this section for any PR awaiting the next release -->

Reply:
I guess that it gets removed when there is something added to the section -- but that's maybe also a good question for the CLI team.